foundations/foundation-common/src/main/java/io/servicecomb/foundation/common/utils/RollingFileAppenderExt.java

```diff
@@ -67,6 +67,10 @@ protected void subAppend(LoggingEvent event) {
     super.subAppend(event);
   }
 
+  public void setMaxBackupIndex(int maxBackups) {
```

Comment:
why need to add this code?
base class already public it.

Reply:
Fixed,removed